README.md

```diff
@@ -776,6 +776,8 @@ scan:
                 [-ask-key-password]
                 [-debug]
                 [-pipe]
+                [-timeout]
+                [-timeout-detect-platform]
```

Comment:
After you summarised it, I guess your are right and options 3 looks good. We should try to minimise number of options to not make CLI to complex and chaotic.